backward.hpp

```diff
@@ -1391,23 +1391,22 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libdw>:
 					if (die_has_pc(die, pc)) {
 						return result;
 					}
-				default:
```

Comment:
what's the warning that requires to move the code out? I do like how it reads when parts of the default clause.